Third-party cookies: update link in D.O.C. (#4724)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

<ApiLifecycle access="ie" />

> This guide covers how to upgrade the Okta Sign-In Widget. The upgrade depends on whether your deployment model is redirect authentication or embedded authentication.

Reviewer: ```suggestion
This guide explains how to upgrade the Okta Sign-In Widget. The upgrade depends on whether your deployment model is redirect authentication or embedded authentication.
```

The widget is a JavaScript library that provides a full-featured and customizable sign-in experience that you can use to authenticate web and mobile app users.

Use the widget on the Okta default sign-in page to do the following:

* Start an Okta SSO session
* Set the Okta [session cookie](/docs/guides/session-cookie/) in the web browser
* Perform a complete [OpenID Connect (OIDC)](/docs/concepts/oauth-openid/) flow
* Integrate with [external Identity Providers](/docs/concepts/identity-providers/)

> >**Note:** Some browsers have begun blocking third-party cookies by default, disrupting Okta functionality in certain flows. See [Mitigate the impact of third-party cookie deprecation](https://help.okta.com/okta_help.htm?type=oie&id=ext-third-party-cookies).

Reviewer: ```suggestion
>**Note:** Some browsers block third-party cookies by default, which disrupts Okta functionality in certain flows. See [Mitigate the impact of third-party cookie deprecation](https://help.okta.com/okta_help.htm?type=oie&id=ext-third-party-cookies).
```

This article teaches you how to upgrade the Sign-In Widget when it’s used in any of the following ways:

> * Redirect authentication (default): Okta provides a sign-in page that is available at your [org's URL](/docs/concepts/okta-organizations/). By default, a user who signs in on this page is redirected to the Okta user dashboard.

Reviewer: ```suggestion
* Redirect authentication (default): Okta provides a sign-in page that's available at your [org's URL](/docs/concepts/okta-organizations/). By default, a user who signs in on this page is redirected to the Okta End-User Dashboard.
```

> * Redirect authentication (customizable): Okta provides a sign-in page that you can customize. You can then make it available under a custom domain that's a subdomain of your company's top-level domain.

Author: ```suggestion
* Redirect authentication (customizable): Okta provides a sign-in page that you can customize. You can then make it available as a custom domain under your company's top-level domain.
```

> * Redirect authentication (customizable): Okta provides a sign-in page that you can customize. You can then make it available under a custom domain that's a subdomain of your company's top-level domain.

Reviewer: Yo dawg, I heard you like custom domains...

> * Embedded authentication (self-hosted): You can embed the widget directly into your application.

Reviewer: ```suggestion
* Embedded authentication (self-hosted): You can embed the widget directly into your app.
```

* `features.idpDiscovery`: Enabled IdP Discovery when the user signed in. The [Identity Provider routing rules](https://help.okta.com/okta_help.htm?id=ext_Identity_Provider_Discovery) are evaluated by default when applicable.

> * `features.showPasswordToggleOnSignInPage`: Provided end users with the ability to toggle the visibility of their password on the Okta sign-in page. This allows users to check their password entry before clicking **Sign In**. This feature also prevented an account lockout due to exceeding their org's permitted number of failed sign-in attempts.

Reviewer: ```suggestion
* `features.showPasswordToggleOnSignInPage`: Provided end users with the ability view their password on the Okta sign-in page. This allows users to check their password entry before clicking **Sign In**. This feature also prevented an account lockout due to exceeding their org's permitted number of failed sign-in attempts.
```
